meson.build

```diff
@@ -45,12 +45,19 @@ endif
 
 sd_journal = dependency('libsystemd-journal', required : false)
 if not sd_journal.found()
-	sd_journal = dependency('libsystemd', required : false)
+  sd_journal = dependency('libsystemd', required : false)
 endif
 if sd_journal.found()
 	add_project_arguments('-DUSE_JOURNALD=1', language : 'c')
 endif
 
+if run_command('./hack/seccomp-notify.sh', check: false).returncode() == 0
```

Comment:
This relies on a shell script to do compile checks, which doesn't respect $CC (there may not necessarily be a `cc` on $PATH, though there likely is) or $PKG_CONFIG (which may point to `pkgconf`).

Maybe it should be ported to something Meson understands natively?

Reply:
That's not something Meson understands natively. Also that doesn't help if someone configures Meson using a machine file (e.g. cross-compilation).